### PR TITLE
add notes to explain the documentation is splited in two : the websit…

### PR DESCRIPTION
**Pull request by maraflush**
Hello,

I would explain the how is splitted the documentation.

Regards,

### CODE REVIEW

**Comment by maraflush** [comment]
Thank for the response:

Are you agree with the following schema ? or I mistaken ?

|Cutter.re repository<br>https://github.org/radareorg/cutter.re|
|:-:|
|^<br>:|
|Cutter.re and blog<br>https://cutter.re|
|^<br>:|
|Cutter<br><br>Documentation|
|:<br>v|
|Technical documentation(api, etc)<b>https://cutter.re/docs|
|:<br>v|
|Cutter repository<br>https://github.org/radareorg/cutter|
|:<br>v|
|Directory docs in the repository<br>https://github.org/radareorg/cutter/docs|

**Comment by ITAYC0HEN** [comment]
Very nice! Thank you for the contribution :)
Here is a little rephrasing of the paragraph, I suggest to use this version:

```
The source for this documentation is available in the `docs directory <https://github.com/radareorg/cutter/tree/master/docs>`_ on Cutter's repository. When the docs are updated, they are generated and pushed directly to the website at https://cutter.re/docs.
The source for the website and blog are available on the `cutter.re repository <http://github.org/radareorg/cutter.re>`_ and are served from the ``gh-pages`` branch.
```

---

As for the location of the paragraphs, I see two good options:

**1. A `note` on above "How can you help?"**
image

---

**2. Adding a paragraph to the "How can you help?" section**
image

**Comment by maraflush** [comment]
I followed your advice, I moved and reworded the sentence in the "How can you help ?"'s section

File: docs/source/contributing/docs/getting-started.rst [hunk 1]
```diff
@@ -16,13 +16,17 @@ How can you help?
 
 The following sections suggest ways you can contribute to Cutter's documentation. The list isn't complete as the possibilities are limitless.
```
Comment (ITAYC0HEN) [inline docs/source/contributing/docs/getting-started.rst:18]: missing "a" on "are" : When the docs are updated
...

File: docs/source/contributing/docs/getting-started.rst [hunk 2]
```diff
@@ -16,13 +16,17 @@ How can you help?
...
+
 
 .. tip::
   Document what you wished to see. If you are a user of Cutter, try to think what things you would want to see documented when you started using the project. Sometimes, the best contributions are coming from your own needs.
 
 User documentation
 ^^^^^^^^^^^^^^^^^^^
 
+
+The source for website and blog are available on the `Cutter.re's repository <http://github.org/radareorg/cutter.re>`_ .
```
Comment (ITAYC0HEN) [inline docs/source/contributing/docs/getting-started.rst:29]: You can give up on this sentence as it is mentioned above

**Comment by maraflush** [comment]
Thank you :)

**Comment by maraflush** [comment]
@Surendrajat do you think the last update is conform at your suggestion ?

**Comment by Surendrajat** [comment]
Looks good.

**Comment by maraflush** [comment]
Thank you

**Pull request merged**